templates/layout.html

```diff
@@ -1,35 +1,90 @@
 <!DOCTYPE html>
```

Comment:
I moved the nav bar out, but I think how I did it is a little sloppy.  Let's talk about the approach to this on Tuesday.